examples/vars.yml

```diff
@@ -35,3 +35,12 @@ lemmyui_env_vars:
 
 postfix_env_vars:
   - POSTFIX_myhostname: "{{ domain }}"
+
+pictrs_safety_env_vars:
+  # Use this in your fedi-safety to allow your worker to authenticate to pictrs-safety
+  - FEDIVERSE_SAFETY_WORKER_AUTH: "{{ lookup('password', 'inventory/host_vars/{{domain}}/passwords/pictrs_safety_worker_auth.psk chars=ascii_letters,digits length=15') }}"
```

Comment:
Please move this to the top of the file like all the other passwords we have. Assign it to a variable, and then have it referenced here.

Also linting wise, we need spaces around domain: `{{ domain }}`.

(If you run ansible-lint against the file it will tell you all the errors)